docs/documentation/cl_options/index.rst

```diff
@@ -0,0 +1,40 @@
+Command Line Arguments
+======================
+
+What starting vimiv form the command line you have the ability to pass a number of
+different argument to vimiv. The general calling structure is depicted below:
+
+.. include:: synopsis.rstsrc
+
+Command Line Arguments
+----------------------
+
+The following is an exhaustive list of all available arguments:
+
+.. include:: options.rstsrc
+
+Examples
+--------
+
+In the following we present a few use cases of command line arguments.
+
```

Comment:
I find the formatting here a bit weird. How about formatting each of the points like this:
```
* Start in thumbnail view instead of image view::

   vimiv * --command "enter thumbnail" --command "enter library"
```
Note that both the number of spaces and the empty line are syntax, restructured text can be a pain :laughing:

Reply:
I agree that the formatting could be better. Though, with your proposed solution double quotes look somewhat odd (probably depending on ones font) and more importantly `--` gets displayed as `-`. In e6a562414448933601ee2b9b8f0000c9fe266fc2 I have mixed yours any my solution. I think this way it looks quite decent. Let me know if you have a better solution.

Reply:
This is how the two versions, the one I proposed and the one in your commit, look for me:

![scrot-2021-05-18_21:46:38](https://user-images.githubusercontent.com/12006766/118713849-bd581f80-b822-11eb-9534-e52602f8d65e.png)

As you can see there is no issue with the double quotes and the `--` characters. Maybe you omitted the two colons? To my understanding the syntax with the two colons is more or less identical to using a `.. code-block`, but I haven't been able to properly indent using an explicit code block :smiley: 

In any case, both options are fine for me, with a slight preference for the gray block.

Reply:
I have indeed missed the two colons :upside_down_face:  It looks much better this was, I have changes it in 5983290df343667c7835c5df4c4db94b0220a9a8

Reply:
Thanks! Restructured text syntax can be a real pain...